src/index.html

```diff
@@ -17,16 +17,35 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form action="/">
+      <div
+        data-qa="big"
+        class="search-bar search-bar--big"
+      >
```

Comment:
This div wrapper is redundant here, you can set this class to form, also your data-qa attribute should be set in form tag

Reply:
done